src/packages/packages.csproj

```diff
@@ -1,7 +1,8 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
Does using the ".Web" project give us a superpset of dependent packages? Subset? Or partially-intersecting set?

Reply:
I believe it's a superset; additionally including the ASP.NET Core libraries